src/templates/resources/header-docs-data.json

```diff
@@ -47,6 +47,12 @@
       "gtmSuffix": "address book",
       "dataTestId": "addressBook",
       "label": "deliveryAddresses"
+    },
+    {
+      "url": "#",
+      "gtmSuffix": "redeem giftcard",
+      "dataTestId": "redeemGiftCard",
+      "label": "redeemGiftcard"
```

Comment:
Hiya I think this change is breaking our global component library repo. You will need to add this attribute to all tenants. Could you update f-header please then bump the package version here: https://github.com/justeat/global-component-library/blob/master/package.json#L24

Error I see is below:

<img width="1002" alt="Screen Shot 2019-06-18 at 4 20 49 PM" src="https://user-images.githubusercontent.com/2299779/59696917-da25eb80-91e4-11e9-92b4-67701bb9ce92.png">